utils/idmloader/scripts/dfn2f90.py

```diff
@@ -29,7 +29,9 @@ def __init__(
         self._warnings = []
         self._multi_package = False
 
-        self.component, self.subcomponent = self._dfnfspec.stem.upper().split("-")
+        self.component, self.subcomponent = self._dfnfspec.stem.upper().split(
```

Comment:
Probably would have been good to have the black changes separate from the changes required for this PR.

Reply:
Moving forward, I won't apply black to .py scripts outside of the autotest foler.  I was assuming all python had been subjected to black.